cms/check_rule_counts

```diff
@@ -50,61 +86,120 @@ if __name__ == '__main__':
                      'SUSPENDED': 'rules_suspend',
                      'WAITING_APPROVAL': 'rules_waiting_approval', }
 
+        # I think both dictionaries in this part of the script are less confusing
+        ages = {
+            'created_24hours_ago': datetime.timedelta(days=1),
+            'created_1week_ago': datetime.timedelta(days=7),
+            'created_3weeks_ago': datetime.timedelta(days=21),
+        }
+
         result = (session.query(models.ReplicationRule.state, func.count(models.ReplicationRule.state))
                   .group_by(models.ReplicationRule.state)
                   .with_hint(models.ReplicationRule, 'INDEX_FFS(rules RULES_PK)', 'oracle')
                   .all())
 
+        print(result)
         for state, num in result:
-            gauge_state = state_map.get(repr(state), 'rules_' + repr(state).lower())
+
+    # Count Rules in x state
+            gauge_state = state_map.get(str(state.name), 'rules_' + str(state.name).lower())
+            #gauge_state = gauge_state.replace("<", "").replace(">", "").split(':')[0]
```

Comment:
Please remove commented out code. No one will have any idea why it's there. Then we'll be ready to merge this one.